src/main/java/pl/agh/edu/master_diet/core/model/rest/browser/activity/ActivitySearchResponse.java

```diff
@@ -0,0 +1,16 @@
+package pl.agh.edu.master_diet.core.model.rest.browser.activity;
+
+import lombok.*;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
+@Builder
```

Comment:
To me, it looks nicer if we arrange annotations from the shortest to longest

Reply:
Done.